aztec/src/main/kotlin/org/wordpress/aztec/formatting/LineBlockFormatter.kt

```diff
@@ -239,6 +238,12 @@ class LineBlockFormatter(editor: AztecText) : AztecFormatter(editor) {
                 position = spanEnd
             }
         }
+        if (position <= 0 && selectionEnd != 0) {
+            position = editableText.indexOf("\n", selectionEnd)
+        }
+        if (position <= 0 && selectionEnd != 0) {
+            position = editableText.length
+        }
```

Comment:
At first I thought the duplicate if-statements were a bug, but after some testing I realized really quickly why this is needed. I think we can simplify this a bit and also make it clear that this behavior is intentional:

```suggestion
        if (position <= 0 && selectionEnd != 0) {
            // If the text contains "\n" return that as the position, else set the position to the end of the text
            position = editableText.indexOf("\n", selectionEnd).takeIf { it >= 0 } ?: editableText.length
        }
```

Reply:
looks good, thanks for the suggestion 👍